### ISSUE TITLE
Merge yoast-social-previews into the JS monorepo

### ISSUE THREAD

**Issue by LisanneKluitmans** [Yoast/javascript#1]
## Summary

The yoast-social-previews repo has to be merged into the javascript monorepo.

`yoast-social-previews` has already been merged in the monorepo. The following still needs to be done before this issue is finished:
- [ ] Have a look at the issues, @atimmer is going to pick that up because it is unclear what needs to happen with them (do we move them, close them because they are outdated or something else).
- [ ] Archive the repository so it is impossible to create new issues or PRs on the repo.

## Explanation

yoast-social-previews, yoastSEO.js, yoast-components, my-yoast client, my-yoast manage and eslint are all merged into the same monorepo to keep all javascript together.

## Technical decisions

All mentioned existing repos above will be placed in their own packages in the monorepo.

**Comment by atimmer** [comment]
I have archived the yoast-social-previews repository. The issues remain open for archival purposes.
